client/src/cmdhf15.c

```diff
@@ -574,6 +574,7 @@ static int CmdHF15Demod(const char *Cmd) {
     for (i = 0; i < 1000; i++) {
         int corr = 0;
         for (j = 0; j < ARRAYLEN(FrameSOF); j += skip) {
+            //TODO Do I really need to use get_graph_value_at()?
```

Comment:
No,   we don't want a "function call" to  make updates to a array in  a loop.    

The solution is a making a local array,   apply changes,   and then copy back to the array we wanted modified...